internal/service/tollgate/controller.go

```diff
@@ -44,18 +44,19 @@ func (c *Controller) QueryOne(ctx context.Context, request *v1beta1.QueryOneRequ
 		}
 		return nil, status.Errorf(codes.Internal, "query tollgate error: %s", err)
 	}
-	return &v1beta1.QueryOneResponse{
+	return &v1beta1.GetTollgateResponse{
 		Tollgate: transform(toll),
 	}, nil
 }
 
-func (c *Controller) QueryAll(ctx context.Context, request *v1beta1.QueryAllRequest) (*v1beta1.QueryAllResponse, error) {
+func (c *Controller) ListTollgates(ctx context.Context, request *v1beta1.ListTollgatesRequest) (*v1beta1.ListTollgatesResponse, error) {
 	all, err := QueryAll(ctx, 100)
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "QueryAll error: %s", err)
 	}
-	return &v1beta1.QueryAllResponse{
-		Tollgates: transformAll(all),
+	return &v1beta1.ListTollgatesResponse{
+		Tollgates:     transformAll(all),
+		NextPageToken: "",
```

Comment:
pagination is coming in later PR?

Reply:
yes, i suggest we have this implemented separately.